notebooks/information_extraction_raven.ipynb

```diff
@@ -0,0 +1,2726 @@
+{
```

Comment:
<p>Can you add an explanation what you're doing with this pipeline? something like "This pipeline will extract the information from the given URL following the provided function arguments"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-cookbook/pull/67/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>